tutorial/source/intro_part_ii.ipynb

```diff
@@ -201,6 +208,27 @@
     "    return pyro.sample(\"weight\", dist.Normal(a, torch.abs(b)))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As an aside, note that in `scale_parametrized_guide`, we had to apply `torch.abs` to parameter `b` because the standard deviation of a normal distribution has to be positive; similar restrictions also apply to parameters of many other distributions. The PyTorch distributions library, which Pyro is built on, includes a [`constraints` module](https://pytorch.org/docs/master/distributions.html#module-torch.distributions.constraints) for enforcing such restrictions, and applying constraints to Pyro parameters is as easy as passing the relevant `constraint` object to `pyro.param`:"
```

Comment:
nit: I believe backticks inside of links breaks one of our docs generators. Could you replace
```diff
- [`constraints` module]
+ [constraints module]
```